test/globals.js

```diff
@@ -1,4 +1,5 @@
 /* @flow */
+import { __ZOID__, __POST_ROBOT__ } from "../globals";
```

Comment:
regarding: https://github.com/paypal/paypal-common-components/actions/runs/8853484326/job/24314427715#step:6:303

Webpack config is using [raw-loader](https://github.com/krakenjs/grumbler-scripts/blob/main/packages/webpack-config-grumbler/index.js#L420), which generates to ESM from that JSON file before webpack uses its own JSON parser on it. We _could_ fix this error by removing JSON from the raw-loader config _or_ by adding `type: javascript/auto` to that rule, which [tells webpack](https://v4.webpack.js.org/configuration/module/#ruletype) to only use the custom loader.

**But** webpack shouldn't be touching `test/globals.js` while running `karma` anyway, so it looks like we can just add `exclude: ['test/globals.js'],` to the [karma config](https://github.com/paypal/paypal-common-components/blob/main/karma.conf.js#L35), and it will skip that file.